Tests/SonarScanner.MSBuild.Shim.Test/ScannerEngineInputGeneratorTest.cs

```diff
@@ -41,11 +42,13 @@ public void Constructor_Null_Throws()
         var cnfg = new AnalysisConfig();
         var rntm = runtime;
         var rvsf = new RoslynV1SarifFixer(runtime.Logger);
-        FluentActions.Invoking(() => new ScannerEngineInputGenerator(null, rntm)).Should().ThrowExactly<ArgumentNullException>().WithParameterName("analysisConfig");
-        FluentActions.Invoking(() => new ScannerEngineInputGenerator(cnfg, null)).Should().ThrowExactly<ArgumentNullException>().WithParameterName("runtime");
-        FluentActions.Invoking(() => new ScannerEngineInputGenerator(cnfg, null, null, null)).Should().ThrowExactly<ArgumentNullException>().WithParameterName("runtime");
-        FluentActions.Invoking(() => new ScannerEngineInputGenerator(cnfg, rntm, null, null)).Should().ThrowExactly<ArgumentNullException>().WithParameterName("fixer");
-        FluentActions.Invoking(() => new ScannerEngineInputGenerator(cnfg, rntm, rvsf, null)).Should().ThrowExactly<ArgumentNullException>().WithParameterName("additionalFilesService");
+        var prvdr = new ListPropertiesProvider();
+        FluentActions.Invoking(() => new ScannerEngineInputGenerator(null, rntm, prvdr)).Should().ThrowExactly<ArgumentNullException>().WithParameterName("analysisConfig");
+        FluentActions.Invoking(() => new ScannerEngineInputGenerator(cnfg, null, prvdr)).Should().ThrowExactly<ArgumentNullException>().WithParameterName("runtime");
+        FluentActions.Invoking(() => new ScannerEngineInputGenerator(cnfg, null, null, null, prvdr)).Should().ThrowExactly<ArgumentNullException>().WithParameterName("runtime");
+        FluentActions.Invoking(() => new ScannerEngineInputGenerator(cnfg, rntm, null, null, prvdr)).Should().ThrowExactly<ArgumentNullException>().WithParameterName("fixer");
+        FluentActions.Invoking(() => new ScannerEngineInputGenerator(cnfg, rntm, rvsf, null, prvdr)).Should().ThrowExactly<ArgumentNullException>().WithParameterName("additionalFilesService");
+        FluentActions.Invoking(() => new ScannerEngineInputGenerator(cnfg, rntm, null)).Should().ThrowExactly<ArgumentNullException>().WithParameterName("cmdLineArgs");
```

Comment:
This should be after line 47, to keep the 3-args together.
And here should be a 5-args with `null` at the end